SPECS/librdkafka/librdkafka.spec

```diff
@@ -87,12 +90,16 @@ DESTDIR=%{buildroot} make install
 %{_libdir}/librdkafka.so
 %{_libdir}/librdkafka++.a
 %{_libdir}/librdkafka++.so
+%{_libdir}/librdkafka-static.a
```

Comment:
Out of curiosity, do we know what the difference is between `librdkafka.a` and `librdkafka-static.a`?

Reply:
https://github.com/confluentinc/librdkafka/blob/e03d3bb91ed92a38f38d9806b8d8deffe78a1de5/packaging/RELEASE.md?plain=1#L222
Our current config doesn't build librdkafka-static.a with external dependencies added. So, the librdkafka.a and librdkafka-static.a files produced are the same (as confirmed by their hash). Excluding it to avoid duplication in RPM thus produced.